docs/guide/known-limitations.md

```diff
@@ -25,96 +25,15 @@ you can't compare the arguments in a formula like this:
-| Coercion of explicit arguments                     | VARP(2, 3, 4, TRUE(), FALSE(), "1",)                                      | 1.9592, based on the behavior of Excel.                                                                                                                                                                                                                                                                | GoogleSheets implementation is not consistent with the standard (see also VAR.S, STDEV.P and STDEV.S function.)                   | 1,9592                                                   |
-| Ranges created with `:`                            | A1:A2<br><br>A$1:$A$2<br><br>A:C<br><br>1:2<br><br>Sheet1!A1:A2           | Allowed ranges consist of two addresses (A1:B5), columns (A:C) or rows (3:5).<br>They cannot be mixed or contain named expressions.                                                                                                                                                                    | Everything allowed.                                                                                                               | Same as Google Sheets.                                   |
-| Formatting inside the TEXT function                | TEXT(A1,"dd-mm-yy")<br><br>TEXT(A1,"###.###”)                             | Not all formatting options are supported,<br>e.g. only some date formatting options: (`hh`, `mm`, `ss`, `am`, `pm`, `a`, `p`, `dd`, `yy`, and `yyyy`).<br><br>No currency formatting inside the TEXT function.                                                                                         | A wide variety of options for string formatting is supported.                                                                     | Same as Google Sheets.                                   |
-| Enabling array arithmetic                          | =ARRAYFORMULA(A2:A5*B2:B5)                                                | By default, array arithmetic is disabled globally.<br><br>The ARRAYFORMULA function enables array arithmetic for the formula inside the function.<br><br>To enable array arithmetic globally, [set the `useArrayArithmetic` option to `true`](../api/interfaces/configparams.html#usearrayarithmetic). | By default, array arithmetic is disabled globally.<br><br>The ARRAYFORMULA function enables array arithmetic inside the function. | By default, array arithmetic is enabled globally.        |
```

Comment:
I removed this row as this issue is described as one of the "configurable incompatibilities"